README.md

```diff
@@ -14,17 +14,18 @@ build
-| RabbitPort                   | The port under which the Rabbit-MQ is reachable                                                 | ""            |
-| RabbitUser                   | The user of the Rabbit-MQ host                                                                  | ""            |
-| RabbitPassword               | The username of the user which connects to the Rabbit-MQ server                                 | ""            |
-| ExchangeName                 | The exchange to which fluent-bit send its logs                                                  | ""            |
-| ExchangeType                 | The exchange-type                                                                               | ""            |
-| RoutingKey                   | The routing-key pattern                                                                         | ""            |
-| RoutingKeyDelimiter          | The Delemiter which seperates the routing-key parts                                             |  "."          |
-| RemoveRkValuesFromRecord     | If enabled fluentd deletes the values of the record,  which have been stored in the routing-key | ""            |
+| Parameter                    | Description                                                                                      | Default-Value |
+|------------------------------|--------------------------------------------------------------------------------------------------|---------------|
+| RabbitHost                   | The hostname of the Rabbit-MQ server                                                             | ""            |
+| RabbitPort                   | The port under which the Rabbit-MQ is reachable                                                  | ""            |
+| RabbitUser                   | The user of the Rabbit-MQ host                                                                   | ""            |
+| RabbitPassword               | The username of the user which connects to the Rabbit-MQ server                                  | ""            |
+| ExchangeName                 | The exchange to which fluent-bit send its logs                                                   | ""            |
+| ExchangeType                 | The exchange-type                                                                                | ""            |
+| RoutingKey                   | The routing-key pattern                                                                          | ""            |
+| RoutingKeyDelimiter          | The Delemiter which seperates the routing-key parts                                              |  "."          |
+| RemoveRkValuesFromRecord     | If enabled fluentd deletes the values of the record,  which have been stored in the routing-key  | ""            |
+| AMQPS                        | If enabled fluent bit will attempt to connecto to RabbitMQ via the amqps protocol instead of amqp| "false"       |
```

Comment:
```suggestion
| AMQPS                        | If enabled fluent bit will attempt to connect to RabbitMQ via the amqps protocol instead of amqp| "false"       |
```